### ISSUE TITLE
base URL in the index.html

### ISSUE THREAD

**Issue by JackKay404** [soedinglab/MMseqs2-App#93]
Would it be possible to add a base URL option in the index.html so that the GUI could be placed behind a reverse proxy? 
Something like here: https://betterstack.com/community/questions/how-to-handle-relative-urls-correctly-with-nginx-reverse-proxy/

**Comment by milot-mirdita** [comment]
This is how we run the various servers. They all run as docker containers with nginx in front (separately from the one in the docker compose). Is this not working for you?

**Comment by JackKay404** [comment]
Absolutely, I understand, and that option works very well for me. However, I would additionally like to run the container on my local server behind a reverse proxy such as Traefik. When I try this the main index.html can be accessed but the hard coded paths in the index.html fail e.g. 
``` 
href="/f-a5df1f3eb4b1c6176a09/apple-touch-icon-72x72.png"
```
From what I understand (and I could be wrong) this could be solved by (optionally) allowing users to set a base href path like in the link I posted above. 

Is that something you would look to add at some point? No worries if it's not the direction you want to take :)
